### PR TITLE
Fix AV1 codecs string

### PR DESCRIPTION
**Pull request by rosetta-jpn**
The correct codecs string for av1 is "av01."

### CODE REVIEW

**Comment by rosetta-jpn** [comment]
I think it is necessary to wait until chrome is fixed. 
https://chromium-review.googlesource.com/c/chromium/src/+/4680672

File: src/content/getusermedia/record/js/main.js [hunk 1]
```diff
@@ -69,11 +69,11 @@ function handleDataAvailable(event) {
 
 function getSupportedMimeTypes() {
   const possibleTypes = [
-    'video/webm;codecs=av1,opus',
     'video/webm;codecs=vp9,opus',
     'video/webm;codecs=vp8,opus',
     'video/webm;codecs=h264,opus',
     'video/mp4;codecs=h264,aac',
+    'video/webm;codecs=av01,opus',
```
Comment (alvestrand) [inline src/content/getusermedia/record/js/main.js:76]: av01 is a fourcc code as specified in RFC 6381 (ANOTHER namespace for codecs).
Is this enough to be correct, or does it need more parameters?

Comment (rosetta-jpn) [reply src/content/getusermedia/record/js/main.js:76]: Looks like it is usual only fourcc is specified in MediaRecorder.

https://www.w3.org/TR/mediastream-recording/#example1

**Comment by fippo** [comment]
I think it is better to show both but add a deprecation notice to the old version

File: src/content/getusermedia/record/js/main.js (hunk 1 above)
Comment (fippo) [reply src/content/getusermedia/record/js/main.js:76]:
```suggestion
    'video/webm;codecs=av01,opus',
    'video/webm;codecs=av1,opus', // deprecated, prefer av01
```

Comment (rosetta-jpn) [reply src/content/getusermedia/record/js/main.js:76]: Done.

**Comment by rosetta-jpn** [comment]
Sorry for not updating long time. I updated my commit.

**Review by fippo — APPROVED** [review]
LGTM. E2E test failures are unrelated to this.

**Comment by rosetta-jpn** [comment]
Thanks fippo for quickly approving and merging, despite of my quite late response!

**Pull request merged**